packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -71,45 +73,47 @@ contract Deploy is Deployer {
     /// @notice Deploy all of the L1 contracts
     function run() public {
         console.log("Deploying a fresh OP Stack including SuperchainConfig");
-        address systemOwnerSafe = deploySafe();
-        address supConfProxy = setupSuperchainConfig();
-        setupOpChain({ supConfProxy: supConfProxy, systemOwnerSafe: systemOwnerSafe });
+
+        address supConfInitiator;
```

Comment:
This should be like:

```solidity
address supConfIgInitiator = cfg.superchainCOnfigInitiator();
if (supConfigInitiator == address(0) {
  supconfigInitator = deploySafe()
}
```

Otherwise supconfInitiator could be `address(0)` when passed to `setupOpChain`